Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.py

```diff
@@ -1757,6 +1839,113 @@ def get_audit_firewall_container_alerts(client: PrismaCloudComputeClient, args:
     )
 
 
+def get_alert_profiles_command(client, args):
+    """
+    Get the alert profiles.
+
+    Args:
+        client (PrismaCloudComputeClient): prisma-cloud-compute client.
+        args (dict): prisma-cloud-compute-get-alert-profiles command arguments
+
+    Returns:
+        CommandResults: command-results object.
+    """
+    project = args.get("project", None)
+    response = client.get_alert_profiles_request(project)
```

Comment:
It will return null when no alert profiles exist

Reply:
in this case we need to handle the case where its `None` otherwise we would fail on `AttributeError` in the next lines, i would do something like:

```suggestion
    response = client.get_alert_profiles_request(project) or []
```